### DISCUSSION TITLE
Overseerr in grafana

### DISCUSSION THREAD

**Discussion by Silastar** [sct/overseerr#1284] [Feature Requests]
Hello!

first of all, I LOVE Overseerr! Thanks for this wonderful addition to my whole Plex infrastructure 

I’m trying to add a panel to my Grafana dashboard that can display pending requests... to no avail... 

I remember Ombi being able to send the info to a panel. 

Is there a way to do so, I would really appreciate some help!

or if it could be added as a json panel

**Comment by ghost** [comment]
Would also love to see the feature, I used a dashboard call "Varken Dashboard" with Ombi.

**Reply by TheCatLady** [reply]
@samwiseg0? 😆

**Reply by samwiseg0** [reply]
This is planned for Varken. I believe there is a PR for support. I just have not gotten around to merging it and I have a few house keeping things to do first. 

https://github.com/Boerderij/Varken/pull/210

**Comment by ikrowni** [comment]
anyone ever get this figured out?
